stroeer/core/v1/article.proto

```diff
@@ -76,3 +89,13 @@ enum AssetType {
   ASSET_TYPE_URL = 3;
   ASSET_TYPE_METADATA = 4;
 }
+
+enum HtmlNodeType {
+  HTML_NODE_TYPE_UNSPECIFIED = 0;
+  HTML_NODE_TYPE_PARAGRAPH = 1;
+  HTML_NODE_TYPE_SUB = 2;
+  HTML_NODE_TYPE_SUP = 3;
+  HTML_NODE_TYPE_ANCHOR = 4;
```

Comment:
der type wird IMHO so im CMS nicht vorkommen

Reply:
oder willst du text zerlegen in seine grundbestandteile?

werden aus diesem Beispiel:
"Hier kommt mein Text. Hat auch einen <a href='www.link.de'>Link</a>. Und auch weiteren Text."

diese pseydo bestandteile?

children:

HTML_NODE_TYPE_PARAGRAPH
text="Hier kommt mein Text. Hat auch einen "

HTML_NODE_TYPE_ANCHOR
text: "Link"
       fields {
         key: "href"
         value: "www.link.de"
       }

HTML_NODE_TYPE_PARAGRAPH
text=". Und auch weiteren Text."

Reply:
externe links? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

Reply:
aber Referenzen sind oft nicht nur ein <a href="...">Text</a> sondern haben vorher und/oder nachher text, mit z.b. Formatierung
<img width="339" alt="Screenshot 2020-08-17 at 15 14 57" src="https://user-images.githubusercontent.com/3657550/90400226-74f5b080-e09c-11ea-9671-f4975980435d.png">

Reply:
dann könnte der ANCHOR halt Kind-Knoten haben, anstelle Text? Etwa so:
```
a {
  text = Werteunion
  children {
    u {
      text = Bayern: Klima-Manifest 2020
    }
    elements {
      themepage {
        fields {
          headline = Themenseite Werteunion
          type = theme_page
        }
        assets {
          teaser_img = http://...
        }
      }
    }
}```

shit in — shit out.